src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx

```diff
@@ -288,20 +294,32 @@ export default function TechnicalIntegration() {
             .map((item) => item)
 
           if (
-            CSVCells[0] === 'roles;description\r' ||
-            CSVCells[0] === 'roles;description'
+            CSVCells[0] === 'roles;description_en;description_de\r' ||
+            CSVCells[0] === 'roles;description_en;description_de'
```

Comment:
The implemented solution wouldn't work with more than two languages in case we provide descriptions for additional languages in the future.